projects/plugins/jetpack/modules/search/instant-search/components/search-filters.scss

```diff
@@ -46,6 +46,7 @@
 		top: 0; // Ensures align-items: center; works as expected.
 
 		// Remove any custom checkbox styling
+		margin: 0;
```

Comment:
This is a nitpick but it looks like the rest of the properties in the file are in alphabetical order - can we move `margin` after `height`?

Reply:
Absolutely, done! ✅